ui/desktop/app/templates/scopes/scope/projects/settings/index.hbs

```diff
@@ -3,4 +3,54 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-{{outlet}}
\ No newline at end of file
+<SettingsCard @header={{t 'titles.application'}} @icon='boundary'>
+  <div class='content'>
+    <div class='app-information'>
+      <div>
+        <Hds::Text::Display>
+          {{t 'titles.version'}}
+        </Hds::Text::Display>
+        <Hds::Text::Body @tag='p' @weight='medium'>
+          {{@model.desktopVersion}}
+        </Hds::Text::Body>
+      </div>
+      <div>
+        <Hds::Text::Display>
+          {{t 'titles.cli'}}
+        </Hds::Text::Display>
+        <Hds::Text::Body @tag='p' @weight='medium'>
+          {{@model.cliVersion}}
+        </Hds::Text::Body>
+      </div>
+      <div>
+        <Hds::Text::Display>
+          {{t 'titles.cache-daemon'}}
+        </Hds::Text::Display>
+        <Hds::Text::Body @tag='p' @weight='medium'>
+          {{#if @model.formattedCacheVersion}}
+            {{@model.formattedCacheVersion}}
+            <Hds::Badge
```

Comment:
Should we display a not running with an icon as well if we don't get the version?

Reply:
updated to show not running and an icon for that